pkg/image/importer/importer.go

```diff
@@ -129,6 +132,11 @@ func (i *ImageStreamImporter) importImages(ctx gocontext.Context, retriever Repo
 		repoName := defaultRef.RepositoryName()
 		registryURL := defaultRef.RegistryURL()
 
+		if !importer.isAllowedRegistry(defaultRef.RegistryHostname()) {
+			isi.Status.Images[i].Status = invalidStatus("", field.Invalid(field.NewPath("from", "name"), from.Name, fmt.Sprintf("registry %q not allowed in master configuration", defaultRef.RegistryHostname())))
```

Comment:
Would be good to demonstrate how this would be shown in oc import-image, oc describe is, web console, etc